src/libraries/System.IO.MemoryMappedFiles/ref/System.IO.MemoryMappedFiles.cs

```diff
@@ -24,6 +24,8 @@ namespace System.IO.MemoryMappedFiles
     public partial class MemoryMappedFile : System.IDisposable
     {
         internal MemoryMappedFile() { }
+        internal MemoryMappedFile(Microsoft.Win32.SafeHandles.SafeMemoryMappedFileHandle handle) { }
```

Comment:
Do we really need internal APIs in ref file?

Reply:
The zero argument one, yeah, since it makes the type not appear to have a default public ctor.  The other two I agree shouldn't be here.

Is this file being hand-edited, vs generated with `dotnet msbuild /t:GenerateReferenceSource` from the src directory (after building the src directory)?

Reply:
It was hand edited. I did not know I had to run that command. I'll do that now.